r2snowman/README.md

```diff
@@ -9,6 +9,6 @@ r2snowman
 - - ----(    . )---\|---
```

Comment:
fix the ascii art
![screenshot 2018-10-25 at 13 41 58](https://user-images.githubusercontent.com/917142/47498112-8b45d300-d85c-11e8-9b34-67e05ec011c7.png)